R/run.R

```diff
@@ -0,0 +1,150 @@
+#' Run multiple Stock Synthesis models
+#'
+#' Loops over a vector of directories and iteratively runs SS in each one
+#'
+#' @param dir Directory, or vector of directories, containing the
+#' model input files.
+#' @template exe
+#' @template extras
+#' @param skipfinished Skip any folders that already contain a Report.sso file.
+#' This can be helpful if the function is interrupted.
+#' @template show_in_console
+#' @param console_output_file File to store console output (if
+#' show_in_console = FALSE)
+#' @template verbose
+#'
+#' @return Returns table showing which directories had model run and which
+#' had errors like missing executable or Report.sso already present.
+#' The three possible messages are "model run failed", "ran model", and
+#' "unknown run status".
+#'
+#' @description First searches path for the specified executable name
```

Comment:
I think this paragraph could use some grammar/clarity edit. e.g., 

```
#' @description The `run` function first searches the PATH for the 
#' specified SS3 executable name
#' and returns the file's location if found. If the SS3 executable is not found in the PATH, 
#' then it is assumed the that executable exists in the specified directory. 
#' Linux systems have an
#' existing executable utility `/usr/sbin/ss` in the path. If `exe =
#' "ss"` and this file is found by this function, it will be ignored
#' based on the smaller file size relative to the SS3 executable. Linux
#' users who want to use the workflow of having SS3 in
#' their PATH should name the SS3 file something besides `ss`, such as `ss3` or
#' `ss_linux`. 

```

(note that this text needs to be re-wrapped)

Reply:
much improved, thank you.

Reply:
also, copying some of this to the @description for check_exe()